exercises/wordy/wordy.t

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env perl6
+
+use Test;
+use JSON::Tiny;
+
+use lib IO::Path.new($?FILE).parent.path;
```

Comment:
No need to change it but just FYI you can do `(my $dir IO::Path.new($?FILE).parent).path` and then reuse the IO::Path object for the directory later :)